apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -93,38 +104,38 @@ export default class SetupChecklist extends Component {
           (isChromeOS() || isChrome()) &&
           !shouldUseWebSerial() &&
           this.detectStep(STATUS_APP_INSTALLED, () =>
-            setupChecker.detectChromeAppInstalled()
+            this.setupChecker.detectChromeAppInstalled()
           )
       )
 
       // Is board plugged in?
       .then(() =>
         this.detectStep(STATUS_BOARD_PLUG, () =>
-          setupChecker.detectBoardPluggedIn()
+          this.setupChecker.detectBoardPluggedIn()
         )
       )
 
       // What type of board is this?
       .then(() => {
-        this.setState({boardTypeDetected: setupChecker.detectBoardType()});
+        this.setState({boardTypeDetected: this.setupChecker.detectBoardType()});
```

Comment:
nit: Call this function once and then use it both places?

```
const boardTypeDetected = this.setupChecker.detectBoardType();
this.setState({ boardTypeDetected });
if (experiments.isEnabled('microbit')) {
  console.log(`Board detected: ${boardTypeDetected}`);
}
```

Reply:
We'll be removing the experimental flag soon (early April) so we'll remove this condition soon.